dogs.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="zxx"> 
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>All Dogs Page</title>
+	<link rel="stylesheet" type="text/css" href="styles/adopt-dog.css">
+</head>
+
+<body>
+	<header>
+		<div class="head-checkout">
+			<div class="logo">
+				<figure>
+					<img src="images/adoptadog-logo.svg" alt="dog LOGO">
+					<figcaption>Seattle’s premiere dog adoption agency</figcaption>
+				</figure>
+			</div>
+			<div class="checkout">
+				<a href="#"><img src="images/cart-icon.svg" alt="checkout image"></a>$1234.56
+				<p>
+				<button type="button">Checkout</button>
+			</div>
+		</div>  <!-- for "head-checkout -->
+	
+		<nav class="clear">
+			<a href="index.html">Home</a> /
+			<a href="dogs.html">Dogs</a> / 
+			<a href="blog.html">Blog</a>
+		</nav>
+	</header>
+
+	<section>
+		<h1>Hare are all of our Dogs! </h1>
+
+		<div class="clear dogs-flex-container">
+			<div>
+				<img src="images/murphy-card.jpg" alt="murphy dog image">
+				<div class="dog">
+					<b>Murphy</b>
```

Comment:
I'd recommend using strong instead of the b tag for bolds. You can set the style via css